/* ______________________________________________________________________
	*
	*	This code is part of the superchick project. 
	*
	* 	Essentially, we are declaring the head controller types
	*  
	*
	*	Author: Olalekan Ogunmolu
	*	Date: Nov. 3, 2016
	*__________________________________________________________________________
*/

namespace amfc_control {
	enum ActuatorType {    
	    base_bladder = 0,
	    left_bladder = 1,
	    right_bladder = 2,
	    total_bladders = 3
	};
}
